platform/xsight/sonic-platform-modules-accton/es9632xx/utils/README

```diff
@@ -0,0 +1,117 @@
+  - The account and password of installed OS can be given at rules/config.
+    The default user and password are "admin" & "YourPaSsWoRd" respectively.
+  - Run "make configure PLATFORM=broadcom"
+  - Copy the built kernel debian package to target/debs/.
+    The file is linux-image-3.16.0-5-amd64_*_amd64.deb under directory
+    src/sonic-linux-kernel/.
+  - Run "make target/sonic-generic.bin"
+  - Get the installer, target/sonic-generic.bin, to target machine and install.
+
+All Linux kernel code is licensed under the GPLv1. All other code is
+licensed under the GPLv3. Please see the LICENSE file for copies of
+both licenses.
+
+The code for integacting with Accton AS7312-54X has 2 parts, 
+kernel drivers and operational script. 
+The kernel drivers of peripherals are under module/ directory.
+1.  These drivers are patched into kernel by 
+    driver-patches-for-accton-as7312-fan-psu-cpld.patch
+    Or you can build the driver under module/ by setting environment variable, 
+    KERNEL_SRC, to proper linux built directory and run make. 
+    It may be sonic-linux-kernel/linux-3.*/debian/build/build_amd64_none_amd64/.
+2.  A operational script, accton_as7312_util.py, for device initializatian and 
+    peripheral accessing should be installed at /usr/bin. 
+    This script is generated by onie_installer-accton-AS7312-54X.patch.
+    It's done by patching onie_installer-accton-AS7312-54X.patch at build-image.
+    Run "accton_as7312_util.py install" to install drivers.
+
+To initialize the system, run "accton_as7312_util.py install".
+To clean up the drivers & devices, run "accton_as7312_util.py clean".
+To dump information of sensors, run "accton_as7312_util.py show".
+To dump SFP EEPROM, run "accton_as7312_util.py sff".
+To set fan speed, run "accton_as7312_util.py set fan".
+To enable/disable SFP emission, run "accton_as7312_util.py set sfp".
+To set system LEDs' color, run "accton_as7312_util.py set led"
+For more information, run "accton_as7312_util.py --help".
+
+====================================================================
+Besides applying accton_as7312_util.py to access peripherals, you can 
+access peripherals by sysfs nodes directly after the installation is run.
+
+System LED:
+    There are 5 system LEDs at the lower-left corner of front panel.
+    They are  loc, diag, fan, ps1, and ps2. 
+    The sysfs interface color mappings are as follows:
+    Brightness:
+        0 => off
+        1 => green
+        2 => amber
+        3 => red
+        4 => blue
+    But not all colors are available for each LED.
+
+Fan Control:
+    There are 10 fans inside 5 fan modules.
+    All fans share 1 duty setting, ranged from 0~100.
+
+Thermal sensers:
+    3 temperature sensors are controlled by the lm75 kernel modules. 
+
+PSUs:
+    There 2 power supplies slot at the left/right side of the back.
+    Once if a PSU is not plugged, the status of it is shown failed.
+
+There are 48 SFP+ and 6 QSFP modules are equipped. 
```

Comment:
This file seems to not be relevant to the platform

Reply:
You are right. we will remove these generic readme files.